contracts/test/CoveragePoolStub.sol

```diff
@@ -1,10 +1,35 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.4;
 
+import "../CoveragePoolConstants.sol";
+
 contract CoveragePoolStub {
+    uint256 public covTotalSupply;
+
     event FundsSeized(address indexed recipient, uint256 portionToSeize);
+    event AssetPoolSharesGranted(address indexed recipient, uint256 covAmount);
 
     function seizeFunds(address recipient, uint256 portionToSeize) external {
         emit FundsSeized(recipient, portionToSeize);
     }
+
+    function grantAssetPoolShares(address recipient, uint256 covAmount)
+        external
+    {
+        emit AssetPoolSharesGranted(recipient, covAmount);
+    }
+
+    function covAmountToGrant(uint256 portionToGrant)
+        external
+        view
+        returns (uint256)
+    {
+        return
+            (portionToGrant * covTotalSupply) /
+            CoveragePoolConstants.FLOATING_POINT_DIVISOR;
+    }
+
+    function setCovTotalSupply(uint256 _covTotalSupply) external {
```

Comment:
Is it used anywhere?

Reply:
Nope, I added it just in case but was not needed eventually. I'll remove.

Reply:
See https://github.com/keep-network/coverage-pools/pull/111/commits/2d11ae71aef5b39ec9329a3ca96e82b002284be4